src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -190,8 +203,16 @@ namespace ts.codefix {
             return { kind: InfoKind.JsxAttributes, token, attributes, parentDeclaration: token.parent };
         }
 
-        if (isIdentifier(token) && isCallExpression(parent)) {
-            return { kind: InfoKind.Function, token, call: parent, sourceFile, modifierFlags: ModifierFlags.None, parentDeclaration: sourceFile };
+        if (isIdentifier(token)) {
+            const type = checker.getContextualType(token);
+            if (type && getObjectFlags(type) & ObjectFlags.Anonymous) {
+                const signature = firstOrUndefined(checker.getSignaturesOfType(type, SignatureKind.Call));

```

Comment:
internal to the checker is getFirstSignatureOfType, although given that you reimplemented it here, it's probably not exported.

Reply:
I didn't find this method in the checker. Maybe it has a different name :(.